Plugins/UdSDK/Source/UdSDK/Private/UDSubsystem.cpp

```diff
@@ -64,16 +64,16 @@ int UUDSubsystem::Init()
 {
 	enum udError error = udE_Success;
 
-	if (const UObjectStorageSettings* Settings = GetDefault<UObjectStorageSettings>())
+	if (const UUDSettings* Settings = GetDefault<UUDSettings>())
 	{
 		ServerUrl = Settings->ServerPath.ToString();
-		APIKey = Settings->Password.ToString();
```

Comment:
The password was being used as the key? What other implications does this change have? Is this change correct because the server is now udCloud?

Reply:
That's correct